README.md

```diff
@@ -147,7 +147,13 @@ Each entry has the following format:
 |         0x8 | `array`    | `size` + `type_code` + payload                         |
 |         0x9 | `map`      | `size` + `key_type_code` + `value_type_code` + payload |
 |         0xA | `row`      | Nested Imprint row (recursive joins)                   |
-|      10–127 | *reserved* | Future primitives / logical types                      |
+|         0xB | `date`     | 4-byte int32 days since Unix epoch                     |
+|         0xC | `time`     | 8-byte int64 milliseconds since midnight               |
+|         0xD | `uuid`     | 16-byte binary representation                          |
+|         0xE | `decimal`  | varint scale + varint length + unscaled bytes          |
+|         0xF | `timestamp`| 8-byte int64 milliseconds since Unix epoch (UTC)      |
```

Comment:
super nitpick but the perfectionist in me is bothered by the fact that `timestamp` and `time` are not one after the other 😆 if it isn't a big deal, it'd be nice to have them both and `date` in the same block